pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -315,7 +315,7 @@ export function show_modal_dialog(props, footerProps) {
             dialogObj.footerProps.dialog_done = closeCallback;
         }
         dialogObj.footerProps.set_error = e => {
-            error = e;
+            error = e === 'object' && e != null ? (e.message || e.toString()) : e;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230202-204107-84adbdf7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)